model/minio/minio_server.rb

```diff
@@ -61,7 +61,7 @@ def init_health_monitor_session
 
   def check_pulse(session:, previous_pulse:)
     reading = begin
-      server_data = JSON.parse(session[:minio_client].admin_info.body)["servers"].find { _1["endpoint"] == "#{vm.ephemeral_net4}:9000" }
+      server_data = JSON.parse(session[:minio_client].admin_info.body)["servers"].find { "http://#{_1["endpoint"]}" == url }
```

Comment:
I don't think this will work after this change https://github.com/ubicloud/ubicloud/pull/1096.

`minio_server.url` returns cluster endpoint without individual indexes for the server. You should use `#{minio_server.hostname}:9000`

Reply:
here is an example return;
``` 
 "network"=>{"minio0.minio.ubicloud.com:9000"=>"online", "minio1.minio.ubicloud.com:9000"=>"online"},
```
Here is the url;
```
=> "http://minio.minio.ubicloud.com:9000"
```
Here is the hostname;
```
=> "minio0.minio.ubicloud.com"
```

Reply:
yes, but that wouldn't work if DNS is not configured. I wanted to have something that would work no matter the DNS' status.

Reply:
Now it's good.